File: Algorithms/Kids_With_the_Greatest_Number_of_Candies/Kids_With_the_Greatest_Number_of_Candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        output = []
        greatest = max(candies)
        
        for x in candies:
            if x + extraCandies >= greatest:
                output.append(True)
            else:
                output.append(False)

        return output

File: Algorithms/Kids_With_the_Greatest_Number_of_Candies/Kids_With_the_Greatest_Number_of_Candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        int greatest = *max_element(candies.begin(),candies.end());
        vector<bool> out;
        
        for(int i = 0; i < candies.size(); i++)
        {
            if(candies[i] + extraCandies >= greatest)
                out.push_back(true);
            else
                out.push_back(false);
        }
        
        return out;
    }
};
